src/constants/routerPaths.js

```diff
@@ -11,6 +11,7 @@ const RouterPaths = {
   makeHera: 'hera',
   testPage: '/test/:id',
   testSolution: '/test-solution/:id',
+  studentDiscipline: '/studentDisciplines/:id',
```

Comment:
i think this is a bad endpoint, because camelcase and the disciplines should have one endpoint controlled by role model